[Bug] pulsar-shell throws massive conscrypt stacktrace for every command when TLS is enabled on arm64

### Search before asking

- [X] I searched in the [issues](https://github.com/apache/pulsar/issues) and found nothing similar.


### Version

I was pleased to find arm64 builds for pulsar 3.0.0, however every administrative tool command leads to a massive stack trace warning that precedes the results of every command, completely munges the output and fills the screen with useless stack-trace information.

Even if the root cause (Conscrypt dependency is broken or architecture-dependent) isn't fixable in the short term, there should be a way to squash this warning, since it makes `pulsar-admin` and `pulshar-shell` _extremely_ unpleasant to use.

### Minimal reproduce step

- Download and install pulsar-shell on arm64.
- Set it up for your cluster with TLS enabled
- Try to use it at all for anything.

### What did you expect to see?

```
tlsclient(pulsar.my.cluster)> admin topics list public/default
persistent://public/default/topic1
persistent://public/default/topic2
tlsclient(pulsar.my.cluster)>
```

### What did you see instead?

```
tlsclient(pulsar.my.cluster)> admin topics list public/default
2023-07-02T00:45:57,946-0400 [main] WARN  org.apache.pulsar.common.util.SecurityUtility - Conscrypt isn't available. Using JDK default security provider.
java.lang.reflect.InvocationTargetException: null
        at jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method) ~[?:?]
        at jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77) ~[?:?]
        at jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43) ~[?:?]
        at java.lang.reflect.Method.invoke(Method.java:568) ~[?:?]
        at org.apache.pulsar.common.util.SecurityUtility.loadConscryptProvider(SecurityUtility.java:125) ~[pulsar-common-2.10.4.2.jar:2.10.4.2]
        at org.apache.pulsar.common.util.SecurityUtility.<clinit>(SecurityUtility.java:81) ~[pulsar-common-2.10.4.2.jar:2.10.4.2]
        at org.apache.pulsar.client.admin.internal.http.AsyncHttpConnector.<init>(AsyncHttpConnector.java:163) ~[pulsar-client-admin-original-2.10.4.2.jar:2.10.4.2]
        at org.apache.pulsar.client.admin.internal.http.AsyncHttpConnectorProvider.getConnector(AsyncHttpConnectorProvider.java:52) ~[pulsar-client-admin-original-2.10.4.2.jar:2.10.4.2]
        at org.apache.pulsar.client.admin.internal.PulsarAdminImpl.<init>(PulsarAdminImpl.java:199) ~[pulsar-client-admin-original-2.10.4.2.jar:2.10.4.2]
        at org.apache.pulsar.client.admin.internal.PulsarAdminBuilderImpl.build(PulsarAdminBuilderImpl.java:49) ~[pulsar-client-admin-original-2.10.4.2.jar:2.10.4.2]
        at org.apache.pulsar.admin.cli.PulsarAdminSupplier.get(PulsarAdminSupplier.java:69) ~[pulsar-client-tools-2.10.4.2.jar:2.10.4.2]
        at org.apache.pulsar.admin.cli.PulsarAdminSupplier.get(PulsarAdminSupplier.java:29) ~[pulsar-client-tools-2.10.4.2.jar:2.10.4.2]
        at org.apache.pulsar.admin.cli.CmdBase.getAdmin(CmdBase.java:112) ~[pulsar-client-tools-2.10.4.2.jar:2.10.4.2]
        at org.apache.pulsar.admin.cli.CmdTopics.getTopics(CmdTopics.java:2876) ~[pulsar-client-tools-2.10.4.2.jar:2.10.4.2]
        at org.apache.pulsar.admin.cli.CmdTopics.access$12200(CmdTopics.java:91) ~[pulsar-client-tools-2.10.4.2.jar:2.10.4.2]
        at org.apache.pulsar.admin.cli.CmdTopics$ListCmd.run(CmdTopics.java:395) ~[pulsar-client-tools-2.10.4.2.jar:2.10.4.2]
        at org.apache.pulsar.admin.cli.CmdBase.run(CmdBase.java:87) ~[pulsar-client-tools-2.10.4.2.jar:2.10.4.2]
        at org.apache.pulsar.admin.cli.PulsarAdminTool.run(PulsarAdminTool.java:257) ~[pulsar-client-tools-2.10.4.2.jar:2.10.4.2]
        at org.apache.pulsar.shell.AdminShell.runCommand(AdminShell.java:72) ~[pulsar-client-tools-2.10.4.2.jar:2.10.4.2]
        at org.apache.pulsar.shell.PulsarShell.run(PulsarShell.java:455) ~[pulsar-client-tools-2.10.4.2.jar:2.10.4.2]
        at org.apache.pulsar.shell.PulsarShell.run(PulsarShell.java:237) ~[pulsar-client-tools-2.10.4.2.jar:2.10.4.2]
        at org.apache.pulsar.shell.PulsarShell.main(PulsarShell.java:215) ~[pulsar-client-tools-2.10.4.2.jar:2.10.4.2]
Caused by: java.lang.UnsatisfiedLinkError: no conscrypt_openjdk_jni-osx-aarch_64 in java.library.path: /Users/awalker/Library/Java/Extensions:/Library/Java/Extensions:/Network/Library/Java/Extensions:/System/Library/Java/Extensions:/usr/lib/java:.
        at java.lang.ClassLoader.loadLibrary(ClassLoader.java:2429) ~[?:?]
        at java.lang.Runtime.loadLibrary0(Runtime.java:818) ~[?:?]
        at java.lang.System.loadLibrary(System.java:1989) ~[?:?]
        at org.conscrypt.NativeLibraryUtil.loadLibrary(NativeLibraryUtil.java:54) ~[conscrypt-openjdk-uber-2.5.2.jar:2.5.2]
        at jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method) ~[?:?]
        at jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77) ~[?:?]
        at jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43) ~[?:?]
        at java.lang.reflect.Method.invoke(Method.java:568) ~[?:?]
        at org.conscrypt.NativeLibraryLoader$1.run(NativeLibraryLoader.java:297) ~[conscrypt-openjdk-uber-2.5.2.jar:2.5.2]
        at org.conscrypt.NativeLibraryLoader$1.run(NativeLibraryLoader.java:289) ~[conscrypt-openjdk-uber-2.5.2.jar:2.5.2]
        at java.security.AccessController.doPrivileged(AccessController.java:318) ~[?:?]
        at org.conscrypt.NativeLibraryLoader.loadLibraryFromHelperClassloader(NativeLibraryLoader.java:289) ~[conscrypt-openjdk-uber-2.5.2.jar:2.5.2]
        at org.conscrypt.NativeLibraryLoader.loadLibrary(NativeLibraryLoader.java:262) ~[conscrypt-openjdk-uber-2.5.2.jar:2.5.2]
        at org.conscrypt.NativeLibraryLoader.load(NativeLibraryLoader.java:162) ~[conscrypt-openjdk-uber-2.5.2.jar:2.5.2]
        at org.conscrypt.NativeLibraryLoader.loadFirstAvailable(NativeLibraryLoader.java:106) ~[conscrypt-openjdk-uber-2.5.2.jar:2.5.2]
        at org.conscrypt.NativeCryptoJni.init(NativeCryptoJni.java:50) ~[conscrypt-openjdk-uber-2.5.2.jar:2.5.2]
        at org.conscrypt.NativeCrypto.<clinit>(NativeCrypto.java:64) ~[conscrypt-openjdk-uber-2.5.2.jar:2.5.2]
        at org.conscrypt.Conscrypt.checkAvailability(Conscrypt.java:119) ~[conscrypt-openjdk-uber-2.5.2.jar:2.5.2]
        ... 22 more
        Suppressed: java.lang.UnsatisfiedLinkError: no conscrypt_openjdk_jni-osx-aarch_64 in java.library.path: /Users/awalker/Library/Java/Extensions:/Library/Java/Extensions:/Network/Library/Java/Extensions:/System/Library/Java/Extensions:/usr/lib/java:.
                at java.lang.ClassLoader.loadLibrary(ClassLoader.java:2429) ~[?:?]
                at java.lang.Runtime.loadLibrary0(Runtime.java:818) ~[?:?]
                at java.lang.System.loadLibrary(System.java:1989) ~[?:?]
                at org.conscrypt.NativeLibraryUtil.loadLibrary(NativeLibraryUtil.java:54) ~[conscrypt-openjdk-uber-2.5.2.jar:2.5.2]
                at org.conscrypt.NativeLibraryLoader.loadLibraryFromCurrentClassloader(NativeLibraryLoader.java:318) ~[conscrypt-openjdk-uber-2.5.2.jar:2.5.2]
                at org.conscrypt.NativeLibraryLoader.loadLibrary(NativeLibraryLoader.java:273) ~[conscrypt-openjdk-uber-2.5.2.jar:2.5.2]
                at org.conscrypt.NativeLibraryLoader.load(NativeLibraryLoader.java:162) ~[conscrypt-openjdk-uber-2.5.2.jar:2.5.2]
                at org.conscrypt.NativeLibraryLoader.loadFirstAvailable(NativeLibraryLoader.java:106) ~[conscrypt-openjdk-uber-2.5.2.jar:2.5.2]
                at org.conscrypt.NativeCryptoJni.init(NativeCryptoJni.java:50) ~[conscrypt-openjdk-uber-2.5.2.jar:2.5.2]
                at org.conscrypt.NativeCrypto.<clinit>(NativeCrypto.java:64) ~[conscrypt-openjdk-uber-2.5.2.jar:2.5.2]
                at org.conscrypt.Conscrypt.checkAvailability(Conscrypt.java:119) ~[conscrypt-openjdk-uber-2.5.2.jar:2.5.2]
                at jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method) ~[?:?]
                at jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77) ~[?:?]
                at jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43) ~[?:?]
                at java.lang.reflect.Method.invoke(Method.java:568) ~[?:?]
                at org.apache.pulsar.common.util.SecurityUtility.loadConscryptProvider(SecurityUtility.java:125) ~[pulsar-common-2.10.4.2.jar:2.10.4.2]
                at org.apache.pulsar.common.util.SecurityUtility.<clinit>(SecurityUtility.java:81) ~[pulsar-common-2.10.4.2.jar:2.10.4.2]
                at org.apache.pulsar.client.admin.internal.http.AsyncHttpConnector.<init>(AsyncHttpConnector.java:163) ~[pulsar-client-admin-original-2.10.4.2.jar:2.10.4.2]
                at org.apache.pulsar.client.admin.internal.http.AsyncHttpConnectorProvider.getConnector(AsyncHttpConnectorProvider.java:52) ~[pulsar-client-admin-original-2.10.4.2.jar:2.10.4.2]
                at org.apache.pulsar.client.admin.internal.PulsarAdminImpl.<init>(PulsarAdminImpl.java:199) ~[pulsar-client-admin-original-2.10.4.2.jar:2.10.4.2]
                at org.apache.pulsar.client.admin.internal.PulsarAdminBuilderImpl.build(PulsarAdminBuilderImpl.java:49) ~[pulsar-client-admin-original-2.10.4.2.jar:2.10.4.2]
                at org.apache.pulsar.admin.cli.PulsarAdminSupplier.get(PulsarAdminSupplier.java:69) ~[pulsar-client-tools-2.10.4.2.jar:2.10.4.2]
                at org.apache.pulsar.admin.cli.PulsarAdminSupplier.get(PulsarAdminSupplier.java:29) ~[pulsar-client-tools-2.10.4.2.jar:2.10.4.2]
                at org.apache.pulsar.admin.cli.CmdBase.getAdmin(CmdBase.java:112) ~[pulsar-client-tools-2.10.4.2.jar:2.10.4.2]
                at org.apache.pulsar.admin.cli.CmdTopics.getTopics(CmdTopics.java:2876) ~[pulsar-client-tools-2.10.4.2.jar:2.10.4.2]
                at org.apache.pulsar.admin.cli.CmdTopics.access$12200(CmdTopics.java:91) ~[pulsar-client-tools-2.10.4.2.jar:2.10.4.2]
                at org.apache.pulsar.admin.cli.CmdTopics$ListCmd.run(CmdTopics.java:395) ~[pulsar-client-tools-2.10.4.2.jar:2.10.4.2]
                at org.apache.pulsar.admin.cli.CmdBase.run(CmdBase.java:87) ~[pulsar-client-tools-2.10.4.2.jar:2.10.4.2]
                at org.apache.pulsar.admin.cli.PulsarAdminTool.run(PulsarAdminTool.java:257) ~[pulsar-client-tools-2.10.4.2.jar:2.10.4.2]
                at org.apache.pulsar.shell.AdminShell.runCommand(AdminShell.java:72) ~[pulsar-client-tools-2.10.4.2.jar:2.10.4.2]
                at org.apache.pulsar.shell.PulsarShell.run(PulsarShell.java:455) ~[pulsar-client-tools-2.10.4.2.jar:2.10.4.2]
                at org.apache.pulsar.shell.PulsarShell.run(PulsarShell.java:237) ~[pulsar-client-tools-2.10.4.2.jar:2.10.4.2]
                at org.apache.pulsar.shell.PulsarShell.main(PulsarShell.java:215) ~[pulsar-client-tools-2.10.4.2.jar:2.10.4.2]
        Suppressed: java.lang.UnsatisfiedLinkError: no conscrypt_openjdk_jni in java.library.path: /Users/awalker/Library/Java/Extensions:/Library/Java/Extensions:/Network/Library/Java/Extensions:/System/Library/Java/Extensions:/usr/lib/java:.
                at java.lang.ClassLoader.loadLibrary(ClassLoader.java:2429) ~[?:?]
                at java.lang.Runtime.loadLibrary0(Runtime.java:818) ~[?:?]
                at java.lang.System.loadLibrary(System.java:1989) ~[?:?]
                at org.conscrypt.NativeLibraryUtil.loadLibrary(NativeLibraryUtil.java:54) ~[conscrypt-openjdk-uber-2.5.2.jar:2.5.2]
                at jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method) ~[?:?]
                at jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77) ~[?:?]
                at jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43) ~[?:?]
                at java.lang.reflect.Method.invoke(Method.java:568) ~[?:?]
                at org.conscrypt.NativeLibraryLoader$1.run(NativeLibraryLoader.java:297) ~[conscrypt-openjdk-uber-2.5.2.jar:2.5.2]
                at org.conscrypt.NativeLibraryLoader$1.run(NativeLibraryLoader.java:289) ~[conscrypt-openjdk-uber-2.5.2.jar:2.5.2]
                at java.security.AccessController.doPrivileged(AccessController.java:318) ~[?:?]
                at org.conscrypt.NativeLibraryLoader.loadLibraryFromHelperClassloader(NativeLibraryLoader.java:289) ~[conscrypt-openjdk-uber-2.5.2.jar:2.5.2]
                at org.conscrypt.NativeLibraryLoader.loadLibrary(NativeLibraryLoader.java:262) ~[conscrypt-openjdk-uber-2.5.2.jar:2.5.2]
                at org.conscrypt.NativeLibraryLoader.load(NativeLibraryLoader.java:162) ~[conscrypt-openjdk-uber-2.5.2.jar:2.5.2]
                at org.conscrypt.NativeLibraryLoader.loadFirstAvailable(NativeLibraryLoader.java:106) ~[conscrypt-openjdk-uber-2.5.2.jar:2.5.2]
                at org.conscrypt.NativeCryptoJni.init(NativeCryptoJni.java:50) ~[conscrypt-openjdk-uber-2.5.2.jar:2.5.2]
                at org.conscrypt.NativeCrypto.<clinit>(NativeCrypto.java:64) ~[conscrypt-openjdk-uber-2.5.2.jar:2.5.2]
                at org.conscrypt.Conscrypt.checkAvailability(Conscrypt.java:119) ~[conscrypt-openjdk-uber-2.5.2.jar:2.5.2]
                at jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method) ~[?:?]
                at jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77) ~[?:?]
                at jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43) ~[?:?]
                at java.lang.reflect.Method.invoke(Method.java:568) ~[?:?]
                at org.apache.pulsar.common.util.SecurityUtility.loadConscryptProvider(SecurityUtility.java:125) ~[pulsar-common-2.10.4.2.jar:2.10.4.2]
                at org.apache.pulsar.common.util.SecurityUtility.<clinit>(SecurityUtility.java:81) ~[pulsar-common-2.10.4.2.jar:2.10.4.2]
                at org.apache.pulsar.client.admin.internal.http.AsyncHttpConnector.<init>(AsyncHttpConnector.java:163) ~[pulsar-client-admin-original-2.10.4.2.jar:2.10.4.2]
                at org.apache.pulsar.client.admin.internal.http.AsyncHttpConnectorProvider.getConnector(AsyncHttpConnectorProvider.java:52) ~[pulsar-client-admin-original-2.10.4.2.jar:2.10.4.2]
                at org.apache.pulsar.client.admin.internal.PulsarAdminImpl.<init>(PulsarAdminImpl.java:199) ~[pulsar-client-admin-original-2.10.4.2.jar:2.10.4.2]
                at org.apache.pulsar.client.admin.internal.PulsarAdminBuilderImpl.build(PulsarAdminBuilderImpl.java:49) ~[pulsar-client-admin-original-2.10.4.2.jar:2.10.4.2]
                at org.apache.pulsar.admin.cli.PulsarAdminSupplier.get(PulsarAdminSupplier.java:69) ~[pulsar-client-tools-2.10.4.2.jar:2.10.4.2]
                at org.apache.pulsar.admin.cli.PulsarAdminSupplier.get(PulsarAdminSupplier.java:29) ~[pulsar-client-tools-2.10.4.2.jar:2.10.4.2]
                at org.apache.pulsar.admin.cli.CmdBase.getAdmin(CmdBase.java:112) ~[pulsar-client-tools-2.10.4.2.jar:2.10.4.2]
                at org.apache.pulsar.admin.cli.CmdTopics.getTopics(CmdTopics.java:2876) ~[pulsar-client-tools-2.10.4.2.jar:2.10.4.2]
                at org.apache.pulsar.admin.cli.CmdTopics.access$12200(CmdTopics.java:91) ~[pulsar-client-tools-2.10.4.2.jar:2.10.4.2]
                at org.apache.pulsar.admin.cli.CmdTopics$ListCmd.run(CmdTopics.java:395) ~[pulsar-client-tools-2.10.4.2.jar:2.10.4.2]
                at org.apache.pulsar.admin.cli.CmdBase.run(CmdBase.java:87) ~[pulsar-client-tools-2.10.4.2.jar:2.10.4.2]
                at org.apache.pulsar.admin.cli.PulsarAdminTool.run(PulsarAdminTool.java:257) ~[pulsar-client-tools-2.10.4.2.jar:2.10.4.2]
                at org.apache.pulsar.shell.AdminShell.runCommand(AdminShell.java:72) ~[pulsar-client-tools-2.10.4.2.jar:2.10.4.2]
                at org.apache.pulsar.shell.PulsarShell.run(PulsarShell.java:455) ~[pulsar-client-tools-2.10.4.2.jar:2.10.4.2]
                at org.apache.pulsar.shell.PulsarShell.run(PulsarShell.java:237) ~[pulsar-client-tools-2.10.4.2.jar:2.10.4.2]
                at org.apache.pulsar.shell.PulsarShell.main(PulsarShell.java:215) ~[pulsar-client-tools-2.10.4.2.jar:2.10.4.2]
        Suppressed: java.lang.UnsatisfiedLinkError: no conscrypt_openjdk_jni in java.library.path: /Users/awalker/Library/Java/Extensions:/Library/Java/Extensions:/Network/Library/Java/Extensions:/System/Library/Java/Extensions:/usr/lib/java:.
                at java.lang.ClassLoader.loadLibrary(ClassLoader.java:2429) ~[?:?]
                at java.lang.Runtime.loadLibrary0(Runtime.java:818) ~[?:?]
                at java.lang.System.loadLibrary(System.java:1989) ~[?:?]
                at org.conscrypt.NativeLibraryUtil.loadLibrary(NativeLibraryUtil.java:54) ~[conscrypt-openjdk-uber-2.5.2.jar:2.5.2]
                at org.conscrypt.NativeLibraryLoader.loadLibraryFromCurrentClassloader(NativeLibraryLoader.java:318) ~[conscrypt-openjdk-uber-2.5.2.jar:2.5.2]
                at org.conscrypt.NativeLibraryLoader.loadLibrary(NativeLibraryLoader.java:273) ~[conscrypt-openjdk-uber-2.5.2.jar:2.5.2]
                at org.conscrypt.NativeLibraryLoader.load(NativeLibraryLoader.java:162) ~[conscrypt-openjdk-uber-2.5.2.jar:2.5.2]
                at org.conscrypt.NativeLibraryLoader.loadFirstAvailable(NativeLibraryLoader.java:106) ~[conscrypt-openjdk-uber-2.5.2.jar:2.5.2]
                at org.conscrypt.NativeCryptoJni.init(NativeCryptoJni.java:50) ~[conscrypt-openjdk-uber-2.5.2.jar:2.5.2]
                at org.conscrypt.NativeCrypto.<clinit>(NativeCrypto.java:64) ~[conscrypt-openjdk-uber-2.5.2.jar:2.5.2]
                at org.conscrypt.Conscrypt.checkAvailability(Conscrypt.java:119) ~[conscrypt-openjdk-uber-2.5.2.jar:2.5.2]
                at jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method) ~[?:?]
                at jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77) ~[?:?]
                at jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43) ~[?:?]
                at java.lang.reflect.Method.invoke(Method.java:568) ~[?:?]
                at org.apache.pulsar.common.util.SecurityUtility.loadConscryptProvider(SecurityUtility.java:125) ~[pulsar-common-2.10.4.2.jar:2.10.4.2]
                at org.apache.pulsar.common.util.SecurityUtility.<clinit>(SecurityUtility.java:81) ~[pulsar-common-2.10.4.2.jar:2.10.4.2]
                at org.apache.pulsar.client.admin.internal.http.AsyncHttpConnector.<init>(AsyncHttpConnector.java:163) ~[pulsar-client-admin-original-2.10.4.2.jar:2.10.4.2]
                at org.apache.pulsar.client.admin.internal.http.AsyncHttpConnectorProvider.getConnector(AsyncHttpConnectorProvider.java:52) ~[pulsar-client-admin-original-2.10.4.2.jar:2.10.4.2]
                at org.apache.pulsar.client.admin.internal.PulsarAdminImpl.<init>(PulsarAdminImpl.java:199) ~[pulsar-client-admin-original-2.10.4.2.jar:2.10.4.2]
                at org.apache.pulsar.client.admin.internal.PulsarAdminBuilderImpl.build(PulsarAdminBuilderImpl.java:49) ~[pulsar-client-admin-original-2.10.4.2.jar:2.10.4.2]
                at org.apache.pulsar.admin.cli.PulsarAdminSupplier.get(PulsarAdminSupplier.java:69) ~[pulsar-client-tools-2.10.4.2.jar:2.10.4.2]
                at org.apache.pulsar.admin.cli.PulsarAdminSupplier.get(PulsarAdminSupplier.java:29) ~[pulsar-client-tools-2.10.4.2.jar:2.10.4.2]
                at org.apache.pulsar.admin.cli.CmdBase.getAdmin(CmdBase.java:112) ~[pulsar-client-tools-2.10.4.2.jar:2.10.4.2]
                at org.apache.pulsar.admin.cli.CmdTopics.getTopics(CmdTopics.java:2876) ~[pulsar-client-tools-2.10.4.2.jar:2.10.4.2]
                at org.apache.pulsar.admin.cli.CmdTopics.access$12200(CmdTopics.java:91) ~[pulsar-client-tools-2.10.4.2.jar:2.10.4.2]
                at org.apache.pulsar.admin.cli.CmdTopics$ListCmd.run(CmdTopics.java:395) ~[pulsar-client-tools-2.10.4.2.jar:2.10.4.2]
                at org.apache.pulsar.admin.cli.CmdBase.run(CmdBase.java:87) ~[pulsar-client-tools-2.10.4.2.jar:2.10.4.2]
                at org.apache.pulsar.admin.cli.PulsarAdminTool.run(PulsarAdminTool.java:257) ~[pulsar-client-tools-2.10.4.2.jar:2.10.4.2]
                at org.apache.pulsar.shell.AdminShell.runCommand(AdminShell.java:72) ~[pulsar-client-tools-2.10.4.2.jar:2.10.4.2]
                at org.apache.pulsar.shell.PulsarShell.run(PulsarShell.java:455) ~[pulsar-client-tools-2.10.4.2.jar:2.10.4.2]
                at org.apache.pulsar.shell.PulsarShell.run(PulsarShell.java:237) ~[pulsar-client-tools-2.10.4.2.jar:2.10.4.2]
                at org.apache.pulsar.shell.PulsarShell.main(PulsarShell.java:215) ~[pulsar-client-tools-2.10.4.2.jar:2.10.4.2]
        Suppressed: java.lang.UnsatisfiedLinkError: no conscrypt in java.library.path: /Users/awalker/Library/Java/Extensions:/Library/Java/Extensions:/Network/Library/Java/Extensions:/System/Library/Java/Extensions:/usr/lib/java:.
                at java.lang.ClassLoader.loadLibrary(ClassLoader.java:2429) ~[?:?]
                at java.lang.Runtime.loadLibrary0(Runtime.java:818) ~[?:?]
                at java.lang.System.loadLibrary(System.java:1989) ~[?:?]
                at org.conscrypt.NativeLibraryUtil.loadLibrary(NativeLibraryUtil.java:54) ~[conscrypt-openjdk-uber-2.5.2.jar:2.5.2]
                at jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method) ~[?:?]
                at jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77) ~[?:?]
                at jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43) ~[?:?]
                at java.lang.reflect.Method.invoke(Method.java:568) ~[?:?]
                at org.conscrypt.NativeLibraryLoader$1.run(NativeLibraryLoader.java:297) ~[conscrypt-openjdk-uber-2.5.2.jar:2.5.2]
                at org.conscrypt.NativeLibraryLoader$1.run(NativeLibraryLoader.java:289) ~[conscrypt-openjdk-uber-2.5.2.jar:2.5.2]
                at java.security.AccessController.doPrivileged(AccessController.java:318) ~[?:?]
                at org.conscrypt.NativeLibraryLoader.loadLibraryFromHelperClassloader(NativeLibraryLoader.java:289) ~[conscrypt-openjdk-uber-2.5.2.jar:2.5.2]
                at org.conscrypt.NativeLibraryLoader.loadLibrary(NativeLibraryLoader.java:262) ~[conscrypt-openjdk-uber-2.5.2.jar:2.5.2]
                at org.conscrypt.NativeLibraryLoader.load(NativeLibraryLoader.java:162) ~[conscrypt-openjdk-uber-2.5.2.jar:2.5.2]
                at org.conscrypt.NativeLibraryLoader.loadFirstAvailable(NativeLibraryLoader.java:106) ~[conscrypt-openjdk-uber-2.5.2.jar:2.5.2]
                at org.conscrypt.NativeCryptoJni.init(NativeCryptoJni.java:50) ~[conscrypt-openjdk-uber-2.5.2.jar:2.5.2]
                at org.conscrypt.NativeCrypto.<clinit>(NativeCrypto.java:64) ~[conscrypt-openjdk-uber-2.5.2.jar:2.5.2]
                at org.conscrypt.Conscrypt.checkAvailability(Conscrypt.java:119) ~[conscrypt-openjdk-uber-2.5.2.jar:2.5.2]
                at jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method) ~[?:?]
                at jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77) ~[?:?]
                at jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43) ~[?:?]
                at java.lang.reflect.Method.invoke(Method.java:568) ~[?:?]
                at org.apache.pulsar.common.util.SecurityUtility.loadConscryptProvider(SecurityUtility.java:125) ~[pulsar-common-2.10.4.2.jar:2.10.4.2]
                at org.apache.pulsar.common.util.SecurityUtility.<clinit>(SecurityUtility.java:81) ~[pulsar-common-2.10.4.2.jar:2.10.4.2]
                at org.apache.pulsar.client.admin.internal.http.AsyncHttpConnector.<init>(AsyncHttpConnector.java:163) ~[pulsar-client-admin-original-2.10.4.2.jar:2.10.4.2]
                at org.apache.pulsar.client.admin.internal.http.AsyncHttpConnectorProvider.getConnector(AsyncHttpConnectorProvider.java:52) ~[pulsar-client-admin-original-2.10.4.2.jar:2.10.4.2]
                at org.apache.pulsar.client.admin.internal.PulsarAdminImpl.<init>(PulsarAdminImpl.java:199) ~[pulsar-client-admin-original-2.10.4.2.jar:2.10.4.2]
                at org.apache.pulsar.client.admin.internal.PulsarAdminBuilderImpl.build(PulsarAdminBuilderImpl.java:49) ~[pulsar-client-admin-original-2.10.4.2.jar:2.10.4.2]
                at org.apache.pulsar.admin.cli.PulsarAdminSupplier.get(PulsarAdminSupplier.java:69) ~[pulsar-client-tools-2.10.4.2.jar:2.10.4.2]
                at org.apache.pulsar.admin.cli.PulsarAdminSupplier.get(PulsarAdminSupplier.java:29) ~[pulsar-client-tools-2.10.4.2.jar:2.10.4.2]
                at org.apache.pulsar.admin.cli.CmdBase.getAdmin(CmdBase.java:112) ~[pulsar-client-tools-2.10.4.2.jar:2.10.4.2]
                at org.apache.pulsar.admin.cli.CmdTopics.getTopics(CmdTopics.java:2876) ~[pulsar-client-tools-2.10.4.2.jar:2.10.4.2]
                at org.apache.pulsar.admin.cli.CmdTopics.access$12200(CmdTopics.java:91) ~[pulsar-client-tools-2.10.4.2.jar:2.10.4.2]
                at org.apache.pulsar.admin.cli.CmdTopics$ListCmd.run(CmdTopics.java:395) ~[pulsar-client-tools-2.10.4.2.jar:2.10.4.2]
                at org.apache.pulsar.admin.cli.CmdBase.run(CmdBase.java:87) ~[pulsar-client-tools-2.10.4.2.jar:2.10.4.2]
                at org.apache.pulsar.admin.cli.PulsarAdminTool.run(PulsarAdminTool.java:257) ~[pulsar-client-tools-2.10.4.2.jar:2.10.4.2]
                at org.apache.pulsar.shell.AdminShell.runCommand(AdminShell.java:72) ~[pulsar-client-tools-2.10.4.2.jar:2.10.4.2]
                at org.apache.pulsar.shell.PulsarShell.run(PulsarShell.java:455) ~[pulsar-client-tools-2.10.4.2.jar:2.10.4.2]
                at org.apache.pulsar.shell.PulsarShell.run(PulsarShell.java:237) ~[pulsar-client-tools-2.10.4.2.jar:2.10.4.2]
                at org.apache.pulsar.shell.PulsarShell.main(PulsarShell.java:215) ~[pulsar-client-tools-2.10.4.2.jar:2.10.4.2]
        Suppressed: java.lang.UnsatisfiedLinkError: no conscrypt in java.library.path: /Users/awalker/Library/Java/Extensions:/Library/Java/Extensions:/Network/Library/Java/Extensions:/System/Library/Java/Extensions:/usr/lib/java:.
                at java.lang.ClassLoader.loadLibrary(ClassLoader.java:2429) ~[?:?]
                at java.lang.Runtime.loadLibrary0(Runtime.java:818) ~[?:?]
                at java.lang.System.loadLibrary(System.java:1989) ~[?:?]
                at org.conscrypt.NativeLibraryUtil.loadLibrary(NativeLibraryUtil.java:54) ~[conscrypt-openjdk-uber-2.5.2.jar:2.5.2]
                at org.conscrypt.NativeLibraryLoader.loadLibraryFromCurrentClassloader(NativeLibraryLoader.java:318) ~[conscrypt-openjdk-uber-2.5.2.jar:2.5.2]
                at org.conscrypt.NativeLibraryLoader.loadLibrary(NativeLibraryLoader.java:273) ~[conscrypt-openjdk-uber-2.5.2.jar:2.5.2]
                at org.conscrypt.NativeLibraryLoader.load(NativeLibraryLoader.java:162) ~[conscrypt-openjdk-uber-2.5.2.jar:2.5.2]
                at org.conscrypt.NativeLibraryLoader.loadFirstAvailable(NativeLibraryLoader.java:106) ~[conscrypt-openjdk-uber-2.5.2.jar:2.5.2]
                at org.conscrypt.NativeCryptoJni.init(NativeCryptoJni.java:50) ~[conscrypt-openjdk-uber-2.5.2.jar:2.5.2]
                at org.conscrypt.NativeCrypto.<clinit>(NativeCrypto.java:64) ~[conscrypt-openjdk-uber-2.5.2.jar:2.5.2]
                at org.conscrypt.Conscrypt.checkAvailability(Conscrypt.java:119) ~[conscrypt-openjdk-uber-2.5.2.jar:2.5.2]
                at jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method) ~[?:?]
                at jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77) ~[?:?]
                at jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43) ~[?:?]
                at java.lang.reflect.Method.invoke(Method.java:568) ~[?:?]
                at org.apache.pulsar.common.util.SecurityUtility.loadConscryptProvider(SecurityUtility.java:125) ~[pulsar-common-2.10.4.2.jar:2.10.4.2]
                at org.apache.pulsar.common.util.SecurityUtility.<clinit>(SecurityUtility.java:81) ~[pulsar-common-2.10.4.2.jar:2.10.4.2]
                at org.apache.pulsar.client.admin.internal.http.AsyncHttpConnector.<init>(AsyncHttpConnector.java:163) ~[pulsar-client-admin-original-2.10.4.2.jar:2.10.4.2]
                at org.apache.pulsar.client.admin.internal.http.AsyncHttpConnectorProvider.getConnector(AsyncHttpConnectorProvider.java:52) ~[pulsar-client-admin-original-2.10.4.2.jar:2.10.4.2]
                at org.apache.pulsar.client.admin.internal.PulsarAdminImpl.<init>(PulsarAdminImpl.java:199) ~[pulsar-client-admin-original-2.10.4.2.jar:2.10.4.2]
                at org.apache.pulsar.client.admin.internal.PulsarAdminBuilderImpl.build(PulsarAdminBuilderImpl.java:49) ~[pulsar-client-admin-original-2.10.4.2.jar:2.10.4.2]
                at org.apache.pulsar.admin.cli.PulsarAdminSupplier.get(PulsarAdminSupplier.java:69) ~[pulsar-client-tools-2.10.4.2.jar:2.10.4.2]
                at org.apache.pulsar.admin.cli.PulsarAdminSupplier.get(PulsarAdminSupplier.java:29) ~[pulsar-client-tools-2.10.4.2.jar:2.10.4.2]
                at org.apache.pulsar.admin.cli.CmdBase.getAdmin(CmdBase.java:112) ~[pulsar-client-tools-2.10.4.2.jar:2.10.4.2]
                at org.apache.pulsar.admin.cli.CmdTopics.getTopics(CmdTopics.java:2876) ~[pulsar-client-tools-2.10.4.2.jar:2.10.4.2]
                at org.apache.pulsar.admin.cli.CmdTopics.access$12200(CmdTopics.java:91) ~[pulsar-client-tools-2.10.4.2.jar:2.10.4.2]
                at org.apache.pulsar.admin.cli.CmdTopics$ListCmd.run(CmdTopics.java:395) ~[pulsar-client-tools-2.10.4.2.jar:2.10.4.2]
                at org.apache.pulsar.admin.cli.CmdBase.run(CmdBase.java:87) ~[pulsar-client-tools-2.10.4.2.jar:2.10.4.2]
                at org.apache.pulsar.admin.cli.PulsarAdminTool.run(PulsarAdminTool.java:257) ~[pulsar-client-tools-2.10.4.2.jar:2.10.4.2]
                at org.apache.pulsar.shell.AdminShell.runCommand(AdminShell.java:72) ~[pulsar-client-tools-2.10.4.2.jar:2.10.4.2]
                at org.apache.pulsar.shell.PulsarShell.run(PulsarShell.java:455) ~[pulsar-client-tools-2.10.4.2.jar:2.10.4.2]
                at org.apache.pulsar.shell.PulsarShell.run(PulsarShell.java:237) ~[pulsar-client-tools-2.10.4.2.jar:2.10.4.2]
                at org.apache.pulsar.shell.PulsarShell.main(PulsarShell.java:215) ~[pulsar-client-tools-2.10.4.2.jar:2.10.4.2]
persistent://public/default/topic1
persistent://public/default/topic2
tlsclient(pulsar.my.cluster)>
```

### Anything else?

```
tlsclient(pulsar.my.cluster)> config view tlsclient
webServiceUrl=https://pulsar.my.cluster:443
brokerServiceUrl=pulsar://pulsar.my.cluster:6650
authPlugin=org.apache.pulsar.client.impl.auth.AuthenticationToken
authParams=file:///Users/me/.config/pulsar/auth_token.txt
tlsAllowInsecureConnection=false
tlsEnableHostnameVerification=false
tlsTrustCertsFilePath=/Users/me/.config/pulsar/mycluster.crt
useKeyStoreTls=false
tlsTrustStoreType=JKS
tlsTrustStorePath=
tlsTrustStorePassword=
```

### Are you willing to submit a PR?

- [ ] I'm willing to submit a PR!

--- a/pulsar-common/src/main/java/org/apache/pulsar/common/util/SecurityUtility.java
+++ b/pulsar-common/src/main/java/org/apache/pulsar/common/util/SecurityUtility.java
@@ -131,7 +131,7 @@ public class SecurityUtility {
                         System.getProperty("os.name"), System.getProperty("os.arch"));
             } else {
                 log.warn("Conscrypt isn't available. Using JDK default security provider."
-                        + " Cause : {}, Problem : {}", e.getCause(), e.getMessage());
+                        + " Cause : {}, Reason : {}", e.getCause(), e.getMessage());
             }
             return null;
         }

